R/zzz.r

```diff
@@ -49,5 +49,12 @@ NULL
 
   find_rtools()
 
+  shims <<- list(
```

Comment:
Does it actually work to assign a value to `shims` when devtools is loaded the normal way (i.e. not with `load_all`)? I would expect that it would be a locked binding, and you would have to use an environment instead of a list.